symplyphysics/laws/electricity/magnetic_induction_of_wire.py

```diff
@@ -0,0 +1,49 @@
+from sympy import (Eq, solve, pi)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    dimensionless
+)
+from sympy.physics.units import magnetic_constant
+
+# Description
+## Let there be an infinite thin conductor. Then the magnetic field created by the current in
```

Comment:
Add geometry of conductor to conditions.

Reply:
> Add geometry of conductor to conditions.

Does it mean just to describe the geometry of the conductor in the description?

Reply:
It seems the law depends on it. So conditions should contain geometry of conductor.

Reply:
> It seems the law depends on it. So conditions should contain geometry of conductor.

Geometry requirements - the wire is infinitely thin. This is the only requirement for geometry. I described this requirement in the description.

Reply:
Yes, but I ask to put it into conditions.